Yona/Yona/Login/LoginViewController.swift

```diff
@@ -85,17 +99,49 @@ extension LoginViewController: KeyboardProtocol {
 
 extension LoginViewController: CodeInputViewDelegate {
     func codeInputView(codeInputView: CodeInputView, didFinishWithCode code: String) {
+//        if loginAttempts == totalAttempts {
+//            let defaults = NSUserDefaults.standardUserDefaults()
+//            defaults.setBool(true, forKey: YonaConstants.nsUserDefaultsKeys.isBlocked)
+//            defaults.synchronize()
+//            self.displayAlertMessage("Login", alertDescription: "Your account has been locked for security reasons, use the reset passcode option to enable our account.")
+//            codeInputView.userInteractionEnabled = false
+//            codeInputView.resignFirstResponder()
+//            errorLabel.hidden = false
+//            errorLabel.text = "Your account has been locked for security reasons, use the reset passcode option to enable our account."
+//        }
+//        else {
+//            loginAttempts += 1
+//        }
+        
+//        if let code = userData[YonaConstants.nsUserDefaultsKeys.pincode] as? String{
+//            NSUserDefaults.standardUserDefaults().setObject(code, forKey: YonaConstants.nsUserDefaultsKeys.pincode)
+//            NSUserDefaults.standardUserDefaults().setObject(self.confirmMobileLink, forKey: YonaConstants.nsUserDefaultsKeys.confirmMobileKeyURL)
         
         
         let passcode = KeychainManager.sharedInstance.getPINCode()
         if code ==  passcode {
             codeInputView.resignFirstResponder()
+            let defaults = NSUserDefaults.standardUserDefaults()
+            defaults.setBool(false, forKey: YonaConstants.nsUserDefaultsKeys.isBlocked)
             if let dashboardStoryboard = R.storyboard.dashboard.dashboardStoryboard {
                 navigationController?.pushViewController(dashboardStoryboard, animated: true)
             }
         } else {
             errorLabel.hidden = false
             codeInputView.clear()
+            if loginAttempts == totalAttempts {
+                let defaults = NSUserDefaults.standardUserDefaults()
+                defaults.setBool(true, forKey: YonaConstants.nsUserDefaultsKeys.isBlocked)
+                defaults.synchronize()
+                self.displayAlertMessage("Login", alertDescription: "Your account has been locked for security reasons, use the reset passcode option to enable our account.")
```

Comment:
I agree, I am waiting for the text message to be given by Yona team.BTW I will replace it. Thanks for pointing.